src/alchemlyb/estimators/mbar_.py

```diff
@@ -92,7 +92,10 @@ def fit(self, u_nk):
                               columns=self.states_,
                               index=self.states_) for i in out]
 
-        (self.delta_f_, self.d_delta_f_, self.theta_) = attrs 
+        (self.delta_f_, self.d_delta_f_, self.theta_) = attrs
```

Comment:
Maybe we want to rename those `attrs` to something else to avoid confusion. `free_energy_differences` ?

Reply:
This could be something that could be improved.
but delta_f_, d_delta_f_ are used in all estimators and on the pymbar module.

Reply:
I meant the use of the name `attrs` on the RHS. Now that we are using `df.attrs`, it is very confusing because these free energy diffs aren't the attrs that we are using for units.